frontend/src/components/processMeasurements/ProcessMeasurementsDetailContent.js

```diff
@@ -47,7 +47,7 @@ const ProcessMeasurementsDetailContent = ({
 
     const isLoading = !isLoaded || processMeasurement.isFetching;
     const title =
-        `Sample Process ${[id, processMeasurement && protocolsByID[processMeasurement.protocol]?.name ].filter(Boolean).join(' - ')}`;
+        `Sample Process Measurement ${[id, processMeasurement && protocolsByID[processMeasurement.protocol]?.name ].filter(Boolean).join(' - ')}`;
```

Comment:
I don't think we should expose process measurement to users... it has no meaning for them. I think maybe differentiate the title using Process and Sample Process (for process measurement).